### PR TITLE
Complete CSS Positioning exercise.

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
Screenshot 2023-07-21 at 11 45 11 AM
Screenshot 2023-07-21 at 11 45 00 AM
Screenshot 2023-07-21 at 11 44 39 AM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.meme > img {
+  width: 100%;
+  position: absolute;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: relative;
+  bottom: 420px;
+  left: 240px;
```
Comment (shawnkost) [inline css-positioning/absolute.css]: You want to use as small offset values as possible. Using `top` & `right` would have been better here

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: relative;
```
Comment (shawnkost) [inline css-positioning/absolute.css]: This should be `position: absolute`

File: css-positioning/absolute.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  bottom: 140px;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+  height: 300px;
+  position: relative;
```
Comment (shawnkost) [inline css-positioning/absolute.css]: Should be `position: absolute`

File: css-positioning/absolute.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  text-transform: uppercase;
+  position: relative;
+  bottom: 140px;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+  height: 300px;
```
Comment (shawnkost) [inline css-positioning/absolute.css:41]: Needs a `width` as well

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Dennis!

**Comment by dennis-pnguyen** [comment]
Screenshot 2023-07-21 at 11 53 43 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**